cv32/sim/Common.mk

```diff
@@ -262,6 +262,18 @@ $(CUSTOM)/dhrystone.elf: $(CUSTOM)/dhrystone.c
 		-I $(RISCV)/riscv32-unknown-elf/include \
 		-L $(RISCV)/riscv32-unknown-elf/lib \
 		-lc -lm -lgcc
+
```

Comment:
Do we need this?   I would have expected the existing `make custom CUSTOM_PROG=<my_fancy_new_test_proj>` target to handle this.

Reply:
This is existing code, only the line feed is new.  The test specific code in this Makefile for hello_world, misalign, illegal, fibonacci, and dhrystone could be removed but then all users need to be aware of the new calling method (make custom CUSTOM_PROG=<my_fancy_new_test_proj>) and the ci script needs updating.